deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/Documentor.java

```diff
@@ -30,20 +36,32 @@
   private static final String CLASS_METHOD_NAME_DELIMITER = "#";
   private static final String PARAM = "@param";
   private static final String RETURN = "@return";
-  @VisibleForTesting static final String DOC_OUTPUT_PROPERTY_NAME = "deepLinkDoc.output";
+  @VisibleForTesting
+  static final String DOC_OUTPUT_PROPERTY_NAME = "deepLinkDoc.output";
+  /**
+   * Known extensions.
+   */
+  private static final Map<String, DocumetationWriter> EXTENSIONS =
+      new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
lets keep it simple - no need to maintain a map of extensions - just hardcode it below